ortools/dotnet/Google.OrTools.csproj.in

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
+    <LangVersion>9.0</LangVersion>
```

Comment:
To be validated.

Reply:
Anything that I can help with?

Reply:
Is 4.5.2 valid for lang version 9 ?

Reply:
valid yes, but not all functionality can be used on 4.5.2. Only the ones that doesn't require runtime support. => https://sergiopedri.medium.com/enabling-and-using-c-9-features-on-older-and-unsupported-runtimes-ce384d8debb

As far as I know if there is usage of one that is not working, it throws a compilation error.